TestClass.cpp

```diff
@@ -0,0 +1,234 @@
+#include <iostream>
+#include <cmath>
+#include <cassert>  
```

Comment:
c style assert included but not used ?  

https://stackoverflow.com/questions/1571340/what-is-the-assert-function

consider c++ style
#include <assert.h>
https://www.geeksforgeeks.org/assertions-cc/

Reply:
Yes.. I was trying a few different things.